receiver/prometheusreceiver/metrics_receiver.go

```diff
@@ -17,13 +17,16 @@ import (
 	"time"
 
 	"github.com/go-kit/log"
+	"github.com/google/cadvisor/version"
```

Comment:
Is this the right import?  Seems odd to depend on cAdvisor...

Reply:
You're right, this should be the Prometheus version package, fixed this